src/dotnet-svcutil/lib/src/Shared/TargetFrameworkHelper.cs

```diff
@@ -15,55 +15,32 @@ internal class TargetFrameworkHelper
     {
         public static ReadOnlyDictionary<Version, Version> NetStandardToNetCoreVersionMap { get; } = new ReadOnlyDictionary<Version, Version>(new SortedDictionary<Version, Version>
          {
-            // Service Model requires netstandard1.3 so it is the minimum version that will work.
-             {new Version("1.3"), new Version("1.0") },
-             {new Version("1.4"), new Version("1.0") },
-             {new Version("1.5"), new Version("1.0") },
-             {new Version("1.6"), new Version("1.0") },
-             {new Version("1.6.1"), new Version("1.1") },
-             {new Version("2.0"), new Version("2.0") }
+             {new Version("2.0"), new Version("2.0") },
+             {new Version("2.1"), new Version("3.1") }
+         });
+
+        public static ReadOnlyDictionary<Version, Version> NetCoreToWCFPackageReferenceVersionMap { get; } = new ReadOnlyDictionary<Version, Version>(new SortedDictionary<Version, Version>
+         {
+             {new Version("2.0"), new Version("2.0") },
+             {new Version("2.1"), new Version("2.0") },
+             {new Version("2.2"), new Version("2.0") },
+             {new Version("3.0"), new Version("2.0") },
+             {new Version("3.1"), new Version("2.0") },
+             {new Version("5.0"), new Version("2.0") },
```

Comment:
If I understand this mapping correctly, this is from Net Core to WCF Client, right?  If so, is the 3.1 and 5.0 line correct?

Reply:
Yes, this mapping is from .NET Core to WCF Client, with the mapping value indicating the WCF Client version as specified in another table. For the 3.1 and 5.0 lines, the tag Version("2.0") corresponds to the 4.10 WCF package. Therefore, the data for these lines is correct.